src/helperFunctions/getUser.ts

```diff
@@ -1,19 +1,17 @@
 import { USER_SELF } from '@/components/constants/url';
 import fetch from '@/helperFunctions/fetch';
-import { toast, ToastTypes } from '@/helperFunctions/toast';
 
 const userData = async () => {
```

Comment:
Can you add the return type here using typescript?
so, anyone using this function can get to know what to expect:
- If success
- If failure

**OR**

Create a separate PR, to use redux toolkit which replaces this function and the `useEffect`.

What's your thought @arpit01923 
cc: @Maheima

Reply:
Issue: https://github.com/Real-Dev-Squad/website-status/issues/492